go.mod

```diff
@@ -3,13 +3,15 @@ module github.com/lucas-clemente/quic-go
 go 1.13
 
 require (
+	github.com/alangpierce/go-forceexport v0.0.0-20160317203124-8f1d6941cd75
```

Comment:
This is only needed for tests, since qtls doesn't need to expose the constructor for the ChaCha AEAD.